spec/models/form/sales/questions/shared_ownership_type_spec.rb

```diff
@@ -50,6 +50,10 @@
         "30" => { "value" => "Shared Ownership - 2021 model lease" },
       })
     end
+
+    it "does not show a guidance_partial" do
+      expect(question.guidance_partial).to eq(nil)
```

Comment:
nothing wrong with this, but just to give you another option which is marginally more semantic, this can be:
`expect(thing).to be_nil`